### PR TITLE
Milestone 2 release candidate

### PR DESCRIPTION
**Pull request by GeorgeJJW**
Changes
- Compile individual efforts of EDA into an unified report.
- Converted repo to a Rproject.

Added

- Added links to survey and dataset on README.md.

### CODE REVIEW

**Review by zxzzhangg — APPROVED** [review]
Everything looks good on me and nice work! The summary part is pretty clear, and thankful to add some current investigation issues.

**Review by sedv8808 — APPROVED** [review]
Thank you so much for putting together all our EDA's and in such a nice way.
Everything looks good to me. Lets just wait for @aylapear approval :)

**Review by aylapear — APPROVED** [review]
Reviewed the compiled EDA. Good to release

**Pull request merged**
